src/services/createOrUpdateGoals.test.js

```diff
@@ -61,23 +64,27 @@ describe('createOrUpdateGoals', () => {
       status: 'Not Started',
     });
 
-    await ObjectiveResource.create({
-      objectiveId: objective.id,
-      userProvidedUrl: 'https://www.test.gov',
-    });
+    await processObjectiveForResourcesById(objective.id, ['https://www.test.gov']);
   });
 
   afterAll(async () => {
     await ObjectiveResource.destroy({
       where: {
         objectiveId: objective.id,
       },
+      individualHooks: true,
+    });
+
+    await ResourceGroups.destroy({
```

Comment:
Presuming that this is related to the bad import and there aren't any resource groups

Reply:
```suggestion
    await Resource.destroy({
```